### PR TITLE
Update onboarding.Rmd

### PR DESCRIPTION
**Pull request by danielibsen**
I have made a first draft of the document. Different items have been added, but they need to be specified with the correct information.

Me and Tinne are making the tables in GoogleDocs and then use https://www.tablesgenerator.com/markdown_tables to generate the table. Once it is done.

### CODE REVIEW

**Comment by lwjohnst86** [comment]
Great job you two! I'll look it over and review it later this week. When will you add the tables?

**Comment by danielibsen** [comment]
> Great job you two! I'll look it over and review it later this week. When will you add the tables?

Tinne will look at it next week, so no rush. And we also want to get input to the table from the wider group. We can also wait with this until everyone have seen the table and given input to the table.

File: onboarding.Rmd [hunk 1]
```diff
@@ -2,17 +2,86 @@
...
+1. Introduce yourselves
+2. Go through the table below
+3. Take a tour around the research facility
+4. Arrange follow-up talk
+
+The purpose of the follow-up talk is to catch up on how you have settled in. If you have any specific questions you can also ask them. As the final thing, you and your buddy will evaluate the introduction. You will be given responsibility to update this website if you have encoutered  any information that was not up to date and you should add any other information you would have liked the page to contain. This is a very important step because it ensures that this material is always current and helps the next new team member.
+
+# SDCA management
+
+![SDCA management](images/sdca-management-structure.jpg)
```
Comment (lwjohnst86) [inline onboarding.Rmd:37]: I've just made a few edits for now. Main thing is I've put the images into this repository so its accessible. You store images in the newly created `images/` folder and refer to them like above: `images/file-name.jpg`

Comment (danielibsen) [reply onboarding.Rmd:37]: That's great. And good to know for next time as well.

**Pull request merged**